### DISCUSSION TITLE
[@types/sparql-http-client]

### DISCUSSION THREAD

**Discussion by eduard-bagdagulyan-vertex** [DefinitelyTyped/DefinitelyTyped#68852] [Issues with a @types package]
The sparql-http-client package is updated recently, removing support for CommonJS modules. 

`Error [ERR_REQUIRE_ESM]: require() of ES Module node_modules/sparql-http-client/StreamClient.js not supported.`

[a comment by an automated account is not shown]

**Comment by tpluscode** [comment]
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69032
